packages/loot-core/src/server/accounts/app.ts

```
@@ -268,9 +268,15 @@
-    const accRow = await db.first('SELECT * FROM accounts WHERE id = ?', [
-      upgradingId,
-    ]);
+    const accRow = await db.first<db.DbAccount>(
+      'SELECT * FROM accounts WHERE id = ?',
+      [upgradingId],
+    );
+
+    if (!accRow) {
+      throw new Error(`Account with ID ${upgradingId} not found.`);
+    }
+        
```

ESLint

[error] Delete `········`

(prettier/prettier)